etl-pipeline/services/grin/grin_client.py

```diff
@@ -44,6 +46,35 @@ def get(self, fragment):
             raise IOError("%s got %s unexpectedly" % (url, response.status_code))
         return response.content
 
+    def convert(self, barcodes):
+        # Ask Google to move some barcodes from the "Available" state to "In-Process"
+        # Response to process will always be a 200 and the content will a table of Barcodes and corresponding Statuses.
+        # For each barcode, a status will be returned of either
+        # "Success", "Already being converted" "Other error", "Not allowed to be downloaded"
+        response = []
+
+        if len(barcodes) >= BATCH_LIMIT:
+            chunked_barcodes = [
+                barcodes[i : i + BATCH_LIMIT]
+                for i in range(0, len(barcodes), BATCH_LIMIT)
+            ]
+        else:
+            chunked_barcodes = [barcodes]
```

Comment:
hmm, should the batch limit handled by the consuming script? I feel like we should keep these client methods small and pure.

Reply:
I think this is fine for now!